tests/test_endgame_tablebase.py

```diff
@@ -76,3 +80,11 @@ def request_fixture(self, request):
 
     def test_et_query_perf(self, request_fixture, fen_string):
         run_perf_analytics(request_fixture.node.name, move_from_et_query, fen_string)
+
+
+class TestLilaEndgameTablebase:
+    def test_lila_dtz_bestmove(self):
+        test_fen = "8/4k3/8/8/8/8/3BB3/3K4 w - - 0 1"
+        lila_bestmove = LilaTablebase.query_bestmove(test_fen)
```

Comment:
Does this function exist now? Or is it `query`

Reply:
woops u are righ